blocks/gmo-campaign-details/gmo-campaign-details.js

```diff
@@ -597,6 +579,13 @@ export default async function decorate(block) {
     block.querySelector('.tab-wrapper').addEventListener('click', (event) => {
         switchTab(event.target);
     })
+    block.querySelector('.back-button').addEventListener('click', (event) => {
```

Comment:
@mdickson-adbe can you investigate this method for back click link https://github.com/hlxsites/adobe-gmo/blob/9f58975b74786cc416d166e22a6c39e44933c3a9/scripts/site-config.js#L305